client/src/app/site/login/components/login-mask/login-mask.component.ts

```diff
@@ -99,6 +103,32 @@ export class LoginMaskComponent extends BaseComponent implements OnInit {
             },
             () => {}
         );
+
+        // Maybe the operator changes and the user is logged in. If so, redirect him and boot OpenSlides.
+        this.operatorSubscription = this.operator.getUserObservable().subscribe(user => {
```

Comment:
I added techniques for that. Perhaps we need them in BaseComponent level.
Check that out, move it from BaseView to Base.

https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/site/base/base-view.ts#L83

Reply:
Yes, a generic collection of subscription would be great. This one is special, because I need to control it more precisly than just unsubscribe at `ngOnDestroy` and have it unsubscribed in another places (lines 111 and 155). So this should be ok here.